src/ioctls/vcpu.rs

```diff
@@ -2375,6 +2543,133 @@ mod tests {
         }
     }
 
+    #[cfg(target_arch = "x86_64")]
+    #[test]
+    fn test_sync_regs() {
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        let mut vcpu = vm.create_vcpu(0).unwrap();
+
+        if kvm.check_extension(Cap::SyncRegs) {
```

Comment:
Setting the sync registers works on every machine, so I think we can run the current test without checking for the extension. My proposal was to check for it only for the other test where we call `run` as well. Same thing applies for the documentation tests. What do you think?

Reply:
Ah, yeah that makes sense. I agree with that. It isn't technically going to be a problem until `run` is called, rather than hinting ahead of time.